src/bussinessLogic/ManageMap.java

```diff
@@ -0,0 +1,100 @@
+package bussinessLogic;
+
+import interfaces.MapManager;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class ManageMap implements MapManager {
+    // the number of letters in the alphabet
+    private final int numberOfLetters = 3;
+    // empty hash map for storing the alphabet and adding the reaction time for each
+    // character
+    LinkedHashMap<String, Integer> charMap = new LinkedHashMap<>();
+    // hash map for storing the number of times a char is displayed (needed for
+    // calculating the average)
+    LinkedHashMap<String, Integer> dividerMap = new LinkedHashMap<>();
+    LinkedHashMap<String, Integer> sortedMap;
+    // current char
+    private char currentChar;
+    // create a string to store the chars when needed as string
+    private String charString;
+
+    /**
+     * creates a new charMap and ads the chars to the map
+     */
+    public void setNewCharMap() {
```

Comment:
The method can be made private and called from the constructor of this class.

Reply:
You're saying to call the methods when the object is instantiated?

Reply:
Yes.